src/backend/apps/event/models.py

```diff
@@ -23,6 +23,12 @@ class Event(BaseModel):
     route_from = models.CharField(max_length=128)
     route_to = models.CharField(max_length=128, blank=True)
 
+    # CARS API info
+    highway_segment_names = models.CharField(max_length=128, blank=True, default='')
```

Comment:
How confident are we in these max_lengths?

Reply:
Most of these new columns have 70~80 characters. I pushed a commit to double the maximum just in case.